clang/test/CheckedCRewriter/uncheckedvars.c

```diff
@@ -0,0 +1,23 @@
+// RUN: cconv-standalone -addcr  %s -- | FileCheck -match-full-lines --check-prefixes="CHECK" %s
+
+int *a = (int*) 1;
+//CHECK: int *a = (int*) 1;
+void b() {
+  //CHECK: void b() {
+  int c = *a;
+}
+
+void c() {
+    int *b = (int*)1;
+    { b; }
+    //CHECK: _Unchecked { b; }
```

Comment:
This is redundant, right? Since `c` is not checked?

Reply:
Yes, now correctly tests for this